simulation-runner/retrieve-agent-ssl-certs.sh

```diff
@@ -73,9 +49,6 @@ scp root@${PE_AGENT}:${HOST_KEY} ./target/ssl/hostkey.pem
 scp root@${PE_AGENT}:${CA_CERT} ./target/ssl/cacert.pem
```

Comment:
Not related to these changes, but I got an error message initially when running this against a Puppet Server 2 install:

~~~~
Copying files from jb-centos7
bash: puppet: command not found
~~~~

I believe the Puppet bin directory isn't put into the root user's shell PATH by default with AIO - not sure if that's intentional?  In any event, if I added the following to my `/root/.bashrc` on my target agent host before running the script...

~~~~bash
export PATH=/opt/puppetlabs/puppet/bin:$PATH
~~~~

... then the script worked okay.

Maybe we should document this setup step if not automate it into the script at some point?